src/CFamily.UnitTests/Analysis/CLangAnalyzerTests.cs

```diff
@@ -285,6 +287,27 @@ public async Task TriggerAnalysisAsync_AnalysisFailsDueToInternalMessage_Notifie
             statusNotifier.VerifyNoOtherCalls();
         }
 
+        [TestMethod]
```

Comment:
This is what I ended up with, I wasn't sure if I needed to test anything ontop of the callback order so please let me know

Reply:
I think that's the only new thing we want to test here.